Makefile

```diff
@@ -0,0 +1,41 @@
+install-tools: ## Install required utilities/tools
+	@command -v pdm > /dev/null || { echo >&2 "pdm is not installed. Installing..."; pip install pdm; }
+
+pdm-lock-check: ## Check that the pdm.lock file is in a good shape
+	pdm lock --check
+
+install-deps: install-tools pdm-lock-check ## Install all required dependencies, according to pdm.lock
+	pdm sync
+
+install-deps-test: install-tools pdm-lock-check ## Install all required dev dependencies, according to pdm.lock
+	pdm sync --dev
+
+update-deps: ## Check pyproject.toml for changes, update the lock file if needed, then sync.
+	pdm install
+	pdm install --dev
+
+check-types: ## Checks type hints in sources
+	mypy --explicit-package-bases --disallow-untyped-calls --disallow-untyped-defs --disallow-incomplete-defs scripts
+
+format: ## Format the code into unified format
+	black scripts
+	ruff check scripts --fix --per-file-ignores=scripts/*:S101
+
+verify: ## Verify the code using various linters
+	black --check scripts
+	ruff check scripts --per-file-ignores=scripts/*:S101
```

Comment:
you're going to want to follow this PR up w/ a openshift/release PR to add a verify CI job, otherwise the format/lint rules aren't going to be enforced on new PRs